code/modules/mob/living/carbon/superior_animal/superior_animal.dm

```diff
@@ -575,66 +533,18 @@
 	// MAKE THIS BETTER. we have SO many goddamn superior mobs that this shit NEEDS to be optimal but i am a goddamn
 	// sophmore in college about to get a goddamn job so im pretty tired of workin on this shit.
 	if(!(ticks_processed%3))
-		if (!AI_inactive)
-			handle_status_effects()
-			update_lying_buckled_and_verb_status()
-		if(!never_stimulate_air)
-			var/datum/gas_mixture/environment = loc.return_air_for_internal_lifeform()
-			var/datum/gas_mixture/breath = environment.remove_volume(BREATH_VOLUME)
-			handle_breath(breath)
-			handle_environment(environment) //it should be pretty safe to move this out of ai inactive if this causes problems.
-			if (can_burrow && bad_environment)
-				evacuate()
-			//Fire handling , not passing the whole list because thats unefficient.
-			handle_fire(environment.gas["oxygen"], loc)
 		// this one in particular im very unhappy about. every 3 ticks, if a superior mob is dead to something that doesnt directly apply damage, it dies. i hate this.
 		handle_regular_status_updates() // we should probably still do this even if we're dead or something
 		ticks_processed = 0
 
-	if (!weakened)
-
-		if(!AI_inactive) //we dont need to handle ai if we're disabled
-			handle_ai()
-			//Speaking
-
-			if(speak_chance && prob(speak_chance))
-				visible_emote(emote_see)
-
-			if (following)
-				if (!target_mob) // Are we following someone and not attacking something?
-					if (stat != DEAD)
-						SSmove_manager.move_to(src, following, follow_distance, move_to_delay) // Follow the mob referenced in 'following' and stand almost next to them.
-			else if (!target_mob && last_followed)
-				SSmove_manager.stop_looping(src)
-				last_followed = null // this exists so we only stop the following once, no need to constantly end our walk
-
-	if(life_cycles_before_sleep)
-		life_cycles_before_sleep--
-		return TRUE
-	if(!(AI_inactive && life_cycles_before_sleep))
-		AI_inactive = TRUE
-
-	if(life_cycles_before_scan)
-		life_cycles_before_scan--
-		return FALSE
-	if(check_surrounding_area(viewRange))
-		activate_ai()
-		life_cycles_before_scan = initial(life_cycles_before_scan)/6 //So it doesn't fall asleep just to wake up the next tick
-		return TRUE
-	life_cycles_before_scan = initial(life_cycles_before_scan)
-	return FALSE
```

Comment:
Was already being called on another Life() parrent. No need to repeat this, since the whole function is now moved up.